docs/source/flow-project.rst

```diff
@@ -215,7 +268,7 @@ As shown before, all *eligible* operations can then be executed with:
 
     ~/my_project $ python project.py run
 
-The status determination operates in serial by default, because typically the overhead costs of using threads/processes are large. However this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
+The status determination operates in series by default, because typically the overhead costs of using threads/processes are large. However this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
```

Comment:
```suggestion
Status is determined sequentially because typically the overhead costs of using threads/processes are large. However, this can be configured by setting a value for the ``flow.status_parallelization`` configuration key.
```